composer.json

```diff
@@ -2,7 +2,18 @@
     "name": "cakephp/cakephp",
     "description": "The CakePHP framework",
     "type": "library",
-    "keywords": ["framework"],
+    "keywords": [
+        "framework",
+        "mvc",
+        "rapid-development",
+        "conventions over configuration",
+        "dry",
+        "orm",
+        "form",
+        "validation",
+        "rest-api",
```

Comment:
That kind of implies then that those are also not justified there, and shouldn't have found their way into the main repo :)